internal/cmd/util/deprecation_test.go

```diff
@@ -0,0 +1,48 @@
+package util_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/dustin/go-humanize"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/hetznercloud/cli/internal/cmd/util"
+)
+
+type testDeprecatable struct {
+	isDeprecated         bool
+	unavailableAfter     time.Time
+	deprecationAnnounced time.Time
+}
+
+func (t testDeprecatable) IsDeprecated() bool {
+	return t.isDeprecated
+}
+
+func (t testDeprecatable) UnavailableAfter() time.Time {
+	return t.unavailableAfter
+}
+
+func (t testDeprecatable) DeprecationAnnounced() time.Time {
+	return t.deprecationAnnounced
+}
+
+func TestDescribeDeprecation(t *testing.T) {
```

Comment:
What do you think about adding another test (or convert to table test) for a not-deprecated resource? That would fully cover the function under test.